app/controllers/transactions.js

```diff
@@ -35,6 +35,19 @@ module.exports = function(app) {
     var group = args.group || {};
     var paymentMethod = args.paymentMethod || {};
 
+    if (transaction.amount > 0 && transaction.txnCurrencyFxRate) {
+      // populate netAmountInGroupCurrency for donations
+        transaction.netAmountInGroupCurrency =
+          Math.round((transaction.amountInTxnCurrency
+            - transaction.platformFeeInTxnCurrency
+            - transaction.hostFeeInTxnCurrency
+            - transaction.paymentProcessorFeeInTxnCurrency)
+          *transaction.txnCurrencyFxRate);
+    } else {
+      // populate netAmountInGroupCurrency for "Add Funds" and Expenses
+      transaction.netAmountInGroupCurrency = transaction.amount*100;
```

Comment:
All donations have an fxRate. If the currencies are same, we populate a '1' for the fxRate.

Stripe only deals with fees (it's own and our platform fee) after fx-conversion. That's the reason, we have to store it this way.